WordPressKitTests/ReaderTopicServiceRemote+InterestsTests.swift

```diff
@@ -68,4 +68,17 @@ class ReaderTopicServiceRemoteInterestsTests: RemoteTestCase, RESTTestable {
         waitForExpectations(timeout: timeout, handler: nil)
     }
 
+    func testPathForTopic() {
+        XCTExpectFailure("This test fails at the moment. The failure will be addressed later")
+
+        XCTAssertEqual(
+            ReaderTopicServiceRemote(wordPressComRestApi: .init(baseUrlString: "https://public-api.wordpress.com")).pathForTopic(slug: "foo"),
+            "https://public-api.wordpress.com/rest/v1.2/read/tags/foo/posts"
+        )
```

Comment:
This assertions fails, because the `baseUrlString` doesn't end with a '/'. But, it really should not fail.

Reply:
Actually, I'll remove this unit test from this PR to make things clearer...

Reply:
Moved to https://github.com/wordpress-mobile/WordPressKit-iOS/pull/690